Logic Problem.md

```diff
@@ -0,0 +1,37 @@
+# Summary
+> >This task examined how different coordination demands and how individual contributations could be combined into a group product. To do this, participants performed anagram and logic tasks.
+
+# References
+>Main paper: https://www.researchgate.net/profile/Glenn-Littlepage/publication/247746082_Effects_of_Group_Size_and_Task_Characteristics_on_Group_Performance_A_Test_of_Steiner%27s_Model/links/59073d83a6fdccd580d5debd/Effects-of-Group-Size-and-Task-Characteristics-on-Group-Performance-A-Test-of-Steiners-Model.pdf
+> Logic Problems: Whipkey, R. L. (1982). Skiing vacations. In Moore, R., Fothstein, E., Reineke, K. (Eds.), The Dell big book of crosswords and pencil puzzles #2 (p. 157). New York, NY: Dell.
+
+
+## Materials for alternative versions of the experiment 
+>Three variations of each type of task was used to prevent subjects in one group from communicating to another group
+
```

Comment:
Study Two repeated the same tasks but with 2 or 10 person groups and with increased encouragement to communicate: Each subject was given 5 colored and 10 white cards for logic problems and 2 colored and 4 white cards for anagram problems. They instructed to deposit a card in a box each time he/she spoke. Subjects started with colored cards and when those were used up they moved to white cards